watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/metab.py

```diff
@@ -0,0 +1,628 @@
+        value(m.fs.costing.LCOM),
+    )
+    m.fs.costing.LCOM_comp.display()
+    print(
+        "LCOW:",
+        value(sum(m.fs.costing.LCOW_comp[c] for c in m.fs.costing.LC_comp)),
+        value(m.fs.costing.LCOW),
+    )
+    m.fs.costing.LCOW_comp.display()
+    print(
+        "LCOCR:",
+        value(sum(m.fs.costing.LCOCR_comp[c] for c in m.fs.costing.LC_comp)),
+        value(m.fs.costing.LCOCR),
+    )
+    m.fs.costing.LCOCR_comp.display()
+
+    return m, results
+
+
+def build():
+    # flowsheet set up
+    m = ConcreteModel()
+    m.db = Database()
+
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+    m.fs.prop = prop_ZO.WaterParameterBlock(
+        default={"solute_list": ["cod", "hydrogen", "methane"]}
+    )
+
+    # unit models
+    m.fs.feed = FeedZO(default={"property_package": m.fs.prop})
+    m.fs.metab_hydrogen = MetabZO(
+        default={
+            "property_package": m.fs.prop,
+            "database": m.db,
+            "process_subtype": "hydrogen",
+        },
+    )
+    m.fs.metab_methane = MetabZO(
+        default={
+            "property_package": m.fs.prop,
+            "database": m.db,
+            "process_subtype": "methane",
+        },
+    )
+    m.fs.product_hydrogen = Product(default={"property_package": m.fs.prop})
+    m.fs.product_methane = Product(default={"property_package": m.fs.prop})
+    m.fs.product_H2O = Product(default={"property_package": m.fs.prop})
+
+    # connections
+    m.fs.s01 = Arc(source=m.fs.feed.outlet, destination=m.fs.metab_hydrogen.inlet)
+    m.fs.s02 = Arc(
+        source=m.fs.metab_hydrogen.treated, destination=m.fs.metab_methane.inlet
+    )
+    m.fs.s03 = Arc(
+        source=m.fs.metab_hydrogen.byproduct, destination=m.fs.product_hydrogen.inlet
+    )
+    m.fs.s04 = Arc(
+        source=m.fs.metab_methane.byproduct, destination=m.fs.product_methane.inlet
+    )
+    m.fs.s05 = Arc(
+        source=m.fs.metab_methane.treated, destination=m.fs.product_H2O.inlet
+    )
+    TransformationFactory("network.expand_arcs").apply_to(m)
+
+    # scaling
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def set_operating_conditions(m):
+    # ---specifications---
+    # feed
+    flow_vol = 3.286e-4 * pyunits.m**3 / pyunits.s
+    conc_mass_cod = 6.76 * pyunits.kg / pyunits.m**3
+
+    m.fs.feed.flow_vol[0].fix(flow_vol)
+    m.fs.feed.conc_mass_comp[0, "cod"].fix(conc_mass_cod)
+    m.fs.feed.properties[0].flow_mass_comp["hydrogen"].fix(1e-8)
+    m.fs.feed.properties[0].flow_mass_comp["methane"].fix(1e-8)
+    solve(m.fs.feed)
+
+    # metab_hydrogen
+    m.fs.metab_hydrogen.load_parameters_from_database(use_default_removal=True)
+
+    # metab_methane
+    m.fs.metab_methane.load_parameters_from_database(use_default_removal=True)
+
+
+def initialize_system(m):
+    seq = SequentialDecomposition()
+    seq.options.tear_set = []
+    seq.options.iterLim = 1
+    seq.run(m, lambda u: u.initialize())
```

Comment:
Should we write a helper function that does this? E.g., `initialize_acyclic_flowsheet`?

Reply:
Yeah that would be good, although is there another word besides acyclic that would be affirmative rather than negative? i.e. something that describes the sequential or straight line structure, rather than NOT cyclic

Reply:
Regardless, it doesn't have too much to do with the PR.